sdk/tables/azure-data-tables/azure/data/tables/_deserialize.py

```diff
@@ -174,3 +211,189 @@ def _extract_etag(response):
         return response.headers.get('etag')
 
     return None
+
+
+def normalize_headers(headers):
```

Comment:
nit: it appears there's some inconsistency in some helper functions (that are not exposed to users) are prefixed with a _ and some not. Would be better if they were all with '_' (within reason)